cmd/orb-api/config.example.yml

```diff
@@ -0,0 +1,26 @@
+listener:
+  bind_addr: "0.0.0.0:8080"
+  tls: 
+    auto:
```

Comment:
This uses autotls when specified and will secure a cert using autocert at the supplied domain name.